src/components/todo/todo-delete-popup.tsx

```diff
@@ -0,0 +1,43 @@
+import { useDeleteTodo } from "@/hooks/query/use-todo";
+import Popup from "../@common/portal/popup";
+import { Dispatch, SetStateAction } from "react";
+
+export default function TodoDeletePopup({
+  todoId,
+  todoTitle,
+  isDeletePopupOpen,
+  setIsDeletePopupOpen,
+}: {
+  todoId: number;
+  todoTitle: string;
+  isDeletePopupOpen: boolean;
+  setIsDeletePopupOpen: Dispatch<SetStateAction<boolean>>;
+}) {
+  const { mutate } = useDeleteTodo();
+  return (
+    <div className="ml-auto">
+      <Popup.Root open={isDeletePopupOpen} onOpenChange={setIsDeletePopupOpen}>
+        <Popup.Trigger className="hidden">삭제하기</Popup.Trigger>
+        <Popup.Content>
+          <div className="flex flex-col items-center justify-center text-base font-medium text-slate-800">
+            <span className="line-clamp-1 mb-10 font-bold">"{todoTitle}"</span>
```

Comment:
`"`를 직접 사용하는 부분에서 린트 에러가 발생하고 있습니다. 다음과 같이 수정해주시면 좋을 것 같습니다.

```suggestion
            <span className="line-clamp-1 mb-10 font-bold">&quot;{todoTitle}&quot;</span>
```